interpreter/cling/lib/Interpreter/Interpreter.cpp

```diff
@@ -1031,6 +1031,31 @@ namespace cling {
     return getDynamicLibraryManager()->lookupLibrary(canonicalFile);
   }
 
+  void Interpreter::replaceEnvVars(std::string &Path){
+
+    std::size_t fpos = Path.find("$");
+
+    while(fpos != std::string::npos) {
+      std::size_t spos = Path.find("/", fpos + 1);
+      std::size_t length;
+
+      if(spos != std::string::npos) // if we found a "/"
+        length = spos - fpos;
+      else // we didn't find any "/"
+        length = Path.size();
+
+      std::string envVar = Path.substr(fpos + 1, length -1); //"HOME"
+      const char* envVar_c = envVar.c_str();
+      char* fullPath_c = getenv(envVar_c); // e.g. "/home/user"
```

Comment:
Can you make `fullPath_c` a `const char*`? Even if `getenv()` returns a `char*`: as you are not modifying it, better mark it as such.